src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -90,7 +110,39 @@ private void OnParseCommand(string value)
         }
 
         /// <summary>
-        /// Converts an hex value to a big integer
+        /// Converts a little-endian hex string to big-endian hex string
+        /// input:  ce616f7f74617e0fc4b805583af2602a238df63f
+        /// output: 0x3ff68d232a60f23a5805b8c40f7e61747f6f61ce
+        /// </summary>
+        /// <param name="hex">Hexadecimal value to be converted</param>
+        /// <returns>Returns null when inputs is not little-endian hex string;
+        /// otherwise, returns the string that represents the converted big-endian hex string.
+        /// </returns>
+        public string? LittleEndianToBigEndian(string hex)
+        {
+            hex = hex.ToLower();
+            if (!new Regex("^([0-9a-f]{2})+$").IsMatch(hex)) return null;
+            return "0x" + hex.HexToBytes().Reverse().ToArray().ToHexString(); ;
+        }
+
+        /// <summary>
+        /// Converts a big-endian hex string to little-endian hex string
+        /// input:  0x3ff68d232a60f23a5805b8c40f7e61747f6f61ce
+        /// output: ce616f7f74617e0fc4b805583af2602a238df63f
+        /// </summary>
+        /// <param name="hex">Hexadecimal value to be converted</param>
+        /// <returns>Returns null when inputs is not big-endian hex string;
+        /// otherwise, returns the string that represents the converted little-endian hex string.
+        /// </returns>
+        public string? BigEndianToLittleEndian(string hex)
+        {
+            hex = hex.ToLower();
```

Comment:
```suggestion
            hex = hex.ToLowerInvariant();
```

Reply:
(or change the regex to be case non sensitive)

Reply:
@shargon 
see https://github.com/neo-project/neo/pull/3204#discussion_r1580189512

Reply:
Already modified